content/en/docs/concepts/webhook.md

```diff
@@ -25,7 +25,7 @@ main functions:
 - [`CustomResourceConversionWebhook`](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion):
   The webhook is also responsible for implementing a conversion over versions
   in the cert-manager `CustomResources` (`cert-manager.io`). This means that
-  multiple API versions can be supported, such as `v1alpha2`, `v1alpha3`
+  multiple API versions can be supported, such as `v1`, `v1beta1`
```

Comment:
I would keep alpha2 in here as that is so widely used we should keep mentioning it